doc/install.md

```diff
@@ -83,12 +83,16 @@ We recommend following the instructions further on to install GMT 6.
 
 PyGMT requires the following libraries to be installed:
 
-- [numpy](https://numpy.org) (>= 1.23)
+- [numpy](https://numpy.org)
```

Comment:
After adopting SPEC 0 (#2863), we will also need to pin pandas and xarray. Removing the pinned version from the installation guides because most users don't care about them. For anyone who cares about the versions, we provide the link to the "compatibility table".